kotsadm/migrations/fixtures/deploy/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM postgres:10.7
 
-ENV POSTGRES_USER=shipcloud
```

Comment:
wont you break our dev environments? not the end of the world...

Reply:
wait, is this used in prod... ?

Reply:
no this is just in dev. and yes it will wipe out local data unfortunately. but it's about time we got rid of this

Reply:
wait, maybe? that'd be super weird, let me check

Reply:
nevermind, it's not